docusaurus/docs/dev-docs/migration/v4-to-v5/breaking-changes/admin-panel-rbac-store-updated.md

```diff
@@ -0,0 +1,61 @@
+---
+title: The admin panel RBAC system has been updated
+description: In Strapi 5, there is no `content-manager_rbacManager` anymore, and the regular permissions system is used instead.
+sidebar_label: content-manager_rbacManager removed
+displayed_sidebar: devDocsMigrationV5Sidebar
+tags:
+ - breaking changes
+ - content manager
+ - RBAC
+---
+
+import Intro from '/docs/snippets/breaking-change-page-intro.md'
+import MigrationIntro from '/docs/snippets/breaking-change-page-migration-intro.md'
+
+# The admin panel RBAC system has been updated
+
+In Strapi 5, the `content-manager_rbacManager`, which is a section of Strapi's redux store for the admin panel, is removed and the regular permissions system is used instead. Additionally, the `useRBAC` hook is updated.
+
+<Intro/>
+
+## Breaking change description
+
+**In Strapi v4**
+
+Permissions are handled with the `content-manager_rbacManager` section of the redux store, like in the following generic example:
+
+```tsx
+const cmPermissions useSelector(state => state['content-manager_rbacManager'])
+```
+
+```tsx
+const { allowedActions } = useRBAC({
+	main: [{ action: 'admin::something.main', subject: null }]
+})
+
+const canMain = allowedActions.canMain
+```
+
+**In Strapi 5**
+
+`content-manager_rbacManager` is removed and the regular permissions system is used instead, which implies the `useRBAC` hook is used differently, as in the following generic example:
+
+```tsx
+const { allowedActions } = useRBAC([
+  { action: 'admin::something.main', subject: null }
+])
+
+const canMain = allowedActions.canMain
+```
+
+## Migration
+
+<MigrationIntro />
+
+### Notes
+
+A new RBAC API is available and users can utilise a middleware system to interact with calls. Additional information can be found in the Contributors Documentation, in the [Fetching permissions](https://contributor.strapi.io/docs/core/admin/permissions/frontend/fetching-permissions) and [Authentication](https://contributor.strapi.io/docs/core/admin/features/authentication) sections.
```

Comment:
@joshuaellis  Not sure if those are the proper pages to link to 🤔

Reply:
Only on the V5 branch at the moment, but this link would be more useful :) 

* https://v5.contributor.strapi.io/exports/classes/StrapiApp#addrbacmiddleware

Reply:
Great! Will it always stay on v5.contributor.strapi.io  or does the team plan to move all v5 contrib. docs to simply contributor.strapi.io at release date?

Reply:
afaik that domain will become regular `contributor.strapi.io` on release :)

Reply:
OK, so I'll add a todo to update the link, unless the engineering team plans to introduce redirections? 🤔

Reply:
There aren't any plans currently